### PR TITLE
#2057 Move enhanced stereochemistry icon to left toolbar

### LINKED ISSUE #2057: Move enhanced stereochemistry icon to left toolbar
**Is your feature request related to a problem? Please describe.**
It is required to move enhanced stereochemistry icon to left toolbar after chain since they are related and users think of the two as being connected

**Describe the solution you'd like**

### PR DESCRIPTION
**Pull request by St-Permiakov**
- Moved enhanced stereochemistry icon to left toolbar
- Fixed enhanced stereochemistry dialog radio buttons not activating properly
Resolves #2057

### CODE REVIEW

File: packages/ketcher-react/src/script/ui/dialog/toolbox/enhancedStereo/enhancedStereo.tsx [hunk 1]
```diff
@@ -53,6 +53,8 @@ const EnhancedStereo: FC<Props> = (props) => {
   const { struct, formState, init, ...rest } = props
   const { result, valid } = formState
 
+  console.log(formState)
```
Comment (epam-nikolai-iakushchenko) [inline packages/ketcher-react/src/script/ui/dialog/toolbox/enhancedStereo/enhancedStereo.tsx:56]: leftover?

File: packages/ketcher-react/src/script/ui/views/toolbars/LeftToolbar/LeftToolbar.tsx [hunk 2]
```diff
@@ -146,6 +146,12 @@ const LeftToolbar = (props: Props) => {
           ]}
         />
 
+        {/* #2057 goes here */}
```
Comment (epam-nikolai-iakushchenko) [inline packages/ketcher-react/src/script/ui/views/toolbars/LeftToolbar/LeftToolbar.tsx:149]: why do we need this comment?

File: packages/ketcher-react/src/script/ui/dialog/toolbox/enhancedStereo/enhancedStereo.tsx (hunk 1 above)
Comment (St-Permiakov) [reply packages/ketcher-react/src/script/ui/dialog/toolbox/enhancedStereo/enhancedStereo.tsx]: Oops. I was sure I removed these. Should've checked better. Thanks!
Fixed.

File: packages/ketcher-react/src/script/ui/views/toolbars/LeftToolbar/LeftToolbar.tsx (hunk 2 above)
Comment (St-Permiakov) [reply packages/ketcher-react/src/script/ui/views/toolbars/LeftToolbar/LeftToolbar.tsx]: Same as above. Removed. Thanks again!

**Review by Nitvex — APPROVED** [review]

**Pull request merged**